build_metal.sh

```diff
@@ -1,67 +1,31 @@
 #!/bin/bash
-: '
-TLDR: Follow the steps outlined below to build metal. For more in-depth information, keep reading
```

Comment:
I think these notes about `create_venv.sh` are important.

Should we move these notes elsewhere, such as `CONTRIBUTING.md` or even a section for devs in `INSTALLING.md`?

Reply:
@tt-rkim there are already notes in INSTALLING.md about the virtual environment.

I want to not have any script documentation inside the script, because anytime we change the script or the CMake infra we will have to update it twice, once internal to the shell script, and once in whatever readme we choose to document things.

If it is important information we should put it in customer facing documentation in the README.md.

Can we take this up as a separate issue?